lms/static/js/commerce/views/receipt_view.js

```diff
@@ -196,7 +226,31 @@ var edx = edx || {};
 
         formatMoney: function (moneyStr) {
             return Number(moneyStr).toFixed(2);
-        }
+        },
+
+        /**
+         * Check whether the payment is for the credit course
+         * or not.
+         *
+         * @param  {object} order Receipt data received from the server
+         * @return {string} String of the provider_id or null.
+         */
+        getCreditProviderId: function (order) {
+            var attribute_values,
+                line = order.lines[0];
+            if (this.useEcommerceApi) {
+                attribute_values = _.filter(line.product.attribute_values, function (attribute) {
+                        return attribute.name === 'credit_provider'
+                    });
+
+                // This method assumes that all items in the order are related to a single course.
+                if (attribute_values.length > 0) {
+                    return attribute_values[0]['value'];
```

Comment:
You should use _.find instead if you only want the first match http://underscorejs.org/#find